### PR TITLE
[HOPSWORKS-2037] Rename map_fun and wrapper_fun to train_fn

### PR DESCRIPTION
**Pull request by robzor92**

### CODE REVIEW

File: hops/experiment.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def launch(map_fun, args_dict=None, name='no-name', local_logdir=False, descript
     >>> def train_nn():
```
Comment (moritzmeister) [inline hops/experiment.py:56]: below (line 135) you also renamed it in the example to `train_fn`. doesn't really matter what it's called but let's be consistent.

**Review by moritzmeister — COMMENTED** [review]
one comment, in some examples you renamed it, in some you didn't. I think we can leave the actual `function` examples with different names.

File: hops/experiment.py (hunk 1 above)
Comment (robzor92) [reply hops/experiment.py:56]: Updated the examples to be consistent

**Review by moritzmeister — APPROVED** [review]
good to go

**Pull request merged**